### ISSUE TITLE
Optimize QCBM with Floq simulation

### ISSUE THREAD

**Issue by TDHTTTT** [QC-UCI/QHack#9]
If there is no better way, we can just do a hyperparameter sweep (varying the circuit depth, learning rate) and find the best combo on Floq.

**Comment by TDHTTTT** [comment]
Please also discuss here about possible ways to take advantage of Floq

**Comment by jcwang111** [comment]
When we have 8 qubits, using the default.qubit simulator and using exact probabilities is still the best way to optimize. Maybe Floq would be useful if we did particle swarm and 26 qubits?

**Comment by TDHTTTT** [comment]
Sure let's try that and even though we don't feel ready to submit a 26 quantum job, we still have some new information based on that experiment.
